### PR TITLE
delay imports in command plugin

### LINKED ISSUE #4949: `DEBUG:urllib3.connectionpool` enabled by default without invoking `conda-build`
### What happened?

when I try to `conda  update --all` I have an error message which is:
```
Collecting package metadata (current_repodata.json): \ DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): repo.anaconda.com:443
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): repo.anaconda.com:443
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): repo.anaconda.com:443
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): repo.anaconda.com:443
| DEBUG:urllib3.connectionpool:https://repo.anaconda.com:443 "GET /pkgs/r/noarch/current_repodata.json HTTP/1.1" 304 0
DEBUG:urllib3.connectionpool:https://repo.anaconda.com:443 "GET /pkgs/main/osx-64/current_repodata.json HTTP/1.1" 200 None
DEBUG:urllib3.connectionpool:https://repo.anaconda.com:443 "GET /pkgs/r/osx-64/current_repodata.json HTTP/1.1" 304 0
DEBUG:urllib3.connectionpool:https://repo.anaconda.com:443 "GET /pkgs/main/noarch/current_repodata.json HTTP/1.1" 200 None
done
Solving environment: done
```

### Conda Details

<details>
<summary><code>conda info</code></summary>

```shell
active environment : base
    active env location : /Users/jacques/opt/anaconda3
            shell level : 1
       user config file : /Users/jacques/.condarc
 populated config files : /Users/jacques/.condarc
          conda version : 23.7.1
    conda-build version : 3.26.0
         python version : 3.8.17.final.0
       virtual packages : __archspec=1=x86_64
                          __osx=10.16=0
                          __unix=0=0
       base environment : /Users/jacques/opt/anaconda3  (writable)
      conda av data dir : /Users/jacques/opt/anaconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /Users/jacques/opt/anaconda3/pkgs
                          /Users/jacques/.conda/pkgs
       envs directories : /Users/jacques/opt/anaconda3/envs
                          /Users/jacques/.conda/envs
               platform : osx-64
             user-agent : conda/23.7.1 requests/2.31.0 CPython/3.8.17 Darwin/22.5.0 OSX/10.16
                UID:GID : 501:20
             netrc file : None
           offline mode : False
```

</details>

<details>
<summary><code>conda config</code></summary>

```shell
==> /Users/jacques/.condarc <==
ssl_verify: True
channel_priority: disabled
channels:
  - defaults

==> envvars <==
allow_softlinks: False

==> cmd_line <==
debug: False
json: False
```

</details>

<details>
<summary><code>conda list</code></summary>

```shell
# packages in environment at /Users/jacques/opt/anaconda3:
#
# Name                    Version                   Build  Channel
_ipyw_jlab_nb_ext_conf    0.1.0            py38hecd8cb5_1    defaults
alabaster                 0.7.12             pyhd3eb1b0_0    defaults
anaconda-client           1.12.0           py38hecd8cb5_0    defaults
anaconda-navigator        2.4.2            py38hecd8cb5_0    defaults
ansiwrap                  0.8.4                    pypi_0    pypi
anyio                     3.6.1              pyhd8ed1ab_1    conda-forge
applaunchservices         0.3.0            py38hecd8cb5_0    defaults
appnope                   0.1.3              pyhd8ed1ab_0    conda-forge
appscript                 1.2.2                    pypi_0    pypi
argon2-cffi               21.3.0             pyhd3eb1b0_0    defaults
argon2-cffi-bindings      21.2.0           py38hca72f7f_0    defaults
arrow                     1.2.3            py38hecd8cb5_1    defaults
astroid                   2.14.2           py38hecd8cb5_0    defaults
asttokens                 2.0.5              pyhd3eb1b0_0    defaults
async-lru                 2.0.2              pyhd8ed1ab_0    conda-forge
atomicwrites              1.4.0                      py_0    defaults
attrs                     22.1.0           py38hecd8cb5_0    defaults
autopep8                  1.6.0              pyhd3eb1b0_1    defaults
babel                     2.11.0           py38hecd8cb5_0    defaults
backcall                  0.2.0              pyhd3eb1b0_0    defaults
backports                 1.1                pyhd3eb1b0_0    defaults
backports.functools_lru_cache 1.6.4              pyhd3eb1b0_0    defaults
backports.tempfile        1.0                pyhd3eb1b0_1    defaults
backports.weakref         1.0.post1                  py_1    defaults
beautifulsoup4            4.12.2           py38hecd8cb5_0    defaults
binaryornot               0.4.4              pyhd3eb1b0_1    defaults
black                     23.3.0           py38hecd8cb5_0    defaults
bleach                    5.0.1              pyhd8ed1ab_0    conda-forge
boltons                   23.0.0           py38hecd8cb5_0    defaults
brotlipy                  0.7.0           py38h9ed2024_1003    defaults
bump2version              1.0.1                    pypi_0    pypi
bumpversion               0.6.0                    pypi_0    pypi
bzip2                     1.0.8                h1de35cc_0    defaults
c-ares                    1.19.0               h6c40b1e_0    defaults
ca-certificates           2023.05.30           hecd8cb5_0    defaults
cachetools                4.2.0                    pypi_0    pypi
cctools                   949.0.1             h9abeeb2_25    defaults
cctools_osx-64            949.0.1             hc7db93f_25    defaults
certifi                   2023.7.22        py38hecd8cb5_0    defaults
cffi                      1.15.1           py38h6c40b1e_3    defaults
chardet                   5.0.0            py38h50d1736_0    conda-forge
charset-normalizer        2.1.1              pyhd8ed1ab_0    conda-forge
click                     8.1.3            py38h50d1736_0    conda-forge
cloudpickle               1.6.0                    pypi_0    pypi
clyent                    1.2.2                    py38_1    defaults
colorama                  0.4.6            py38hecd8cb5_0    defaults
comm                      0.1.2            py38hecd8cb5_0    defaults
conda                     23.7.1           py38hecd8cb5_0    defaults
conda-build               3.26.0           py38hecd8cb5_0    defaults
conda-content-trust       0.1.1              pyhd3eb1b0_0    defaults
conda-env                 2.6.0                         1    defaults
conda-index               0.2.3            py38hecd8cb5_0    defaults
conda-package-handling    2.1.0            py38hecd8cb5_0    defaults
conda-package-streaming   0.8.0            py38hecd8cb5_0    defaults
conda-repo-cli            1.0.41           py38hecd8cb5_0    defaults
conda-token               0.4.0              pyhd3eb1b0_0    defaults
conda-verify              3.4.2                      py_1    defaults
configparser              5.0.1                    pypi_0    pypi
cookiecutter              1.7.3              pyhd3eb1b0_0    defaults
coverage                  5.3.1                    pypi_0    pypi
cryptography              41.0.2           py38h2093b18_0    defaults
curl                      8.1.1                h6c40b1e_0    defaults
debugpy                   1.6.3            py38hd331d03_0    conda-forge
decorator                 5.1.1              pyhd3eb1b0_0    defaults
defusedxml                0.7.1              pyhd3eb1b0_0    defaults
deprecated                1.2.11                   pypi_0    pypi
diff-match-patch          20200713           pyhd3eb1b0_0    defaults
dill                      0.3.6            py38hecd8cb5_0    defaults
distlib                   0.3.1                    pypi_0    pypi
docstring-parser          0.7.3                    pypi_0    pypi
docstring-to-markdown     0.11             py38hecd8cb5_0    defaults
docutils                  0.18.1           py38hecd8cb5_3    defaults
elyra-code-snippet-extension 1.5.3                    pypi_0    pypi
elyra-server              1.5.3                    pypi_0    pypi
entrypoints               0.4              py38hecd8cb5_0    defaults
executing                 0.8.3              pyhd3eb1b0_0    defaults
expat                     2.4.9                he9d5cce_0    defaults
filelock                  3.9.0            py38hecd8cb5_0    defaults
flake8                    6.0.0            py38hecd8cb5_0    defaults
freetype                  2.12.1               hd8bbffd_0    defaults
future                    0.18.3           py38hecd8cb5_0    defaults
gdbm                      1.18                 hdccc71a_4    defaults
gettext                   0.21.0               he85b6c0_1    defaults
giflib                    5.2.1                h6c40b1e_3    defaults
git                       2.40.1          pl5340h9355663_1    defaults
gitdb                     4.0.5                    pypi_0    pypi
gitpython                 3.1.12                   pypi_0    pypi
glib                      2.69.1               hfff2838_2    defaults
glob2                     0.7                pyhd3eb1b0_0    defaults
google-api-core           1.25.0                   pypi_0    pypi
google-auth               1.24.0                   pypi_0    pypi
google-cloud-core         1.5.0                    pypi_0    pypi
google-cloud-storage      1.35.0                   pypi_0    pypi
google-crc32c             1.1.1                    pypi_0    pypi
google-resumable-media    1.2.0                    pypi_0    pypi
googleapis-common-protos  1.52.0                   pypi_0    pypi
gst-plugins-base          1.14.1               hcec6c5f_1    defaults
gstreamer                 1.14.1               h6c40b1e_1    defaults
icu                       58.2                 h0a44026_3    defaults
idna                      3.4              py38hecd8cb5_0    defaults
imagesize                 1.4.1            py38hecd8cb5_0    defaults
importlib-metadata        6.0.0            py38hecd8cb5_0    defaults
importlib_metadata        6.0.0                hd3eb1b0_0    defaults
importlib_resources       5.9.0              pyhd8ed1ab_0    conda-forge
inflection                0.5.1            py38hecd8cb5_0    defaults
intervaltree              3.1.0              pyhd3eb1b0_0    defaults
ipykernel                 6.19.2           py38h01d92e1_0    defaults
ipython                   8.12.0           py38hecd8cb5_0    defaults
ipython_genutils          0.2.0              pyhd3eb1b0_1    defaults
ipywidgets                8.0.4            py38hecd8cb5_0    defaults
isort                     5.9.3              pyhd3eb1b0_0    defaults
jaraco.classes            3.2.1              pyhd3eb1b0_0    defaults
jedi                      0.18.1           py38hecd8cb5_1    defaults
jellyfish                 0.9.0            py38hca72f7f_0    defaults
jinja2                    3.1.2            py38hecd8cb5_0    defaults
jinja2-time               0.2.0              pyhd3eb1b0_3    defaults
jpeg                      9e                   h6c40b1e_1    defaults
json5                     0.9.6              pyhd3eb1b0_0    defaults
jsonpatch                 1.32               pyhd3eb1b0_0    defaults
jsonpointer               2.1                pyhd3eb1b0_0    defaults
jsonschema                4.17.3           py38hecd8cb5_0    defaults
jupyter-kite              1.2.0                    pypi_0    pypi
jupyter-lsp               2.2.0              pyhd8ed1ab_0    conda-forge
jupyter-server            1.1.4                    pypi_0    pypi
jupyter_client            8.1.0            py38hecd8cb5_0    defaults
jupyter_core              5.3.0            py38hecd8cb5_0    defaults
jupyter_events            0.6.3            py38hecd8cb5_0    defaults
jupyter_server            2.5.0            py38hecd8cb5_0    defaults
jupyter_server_terminals  0.4.4            py38hecd8cb5_1    defaults
jupyterlab                2.2.9                    pypi_0    pypi
jupyterlab-autoversion    0.3.0                    pypi_0    pypi
jupyterlab-code-formatter 1.4.1                    pypi_0    pypi
jupyterlab-git            0.23.1                   pypi_0    pypi
jupyterlab_pygments       0.2.2              pyhd8ed1ab_0    conda-forge
jupyterlab_server         2.22.0           py38hecd8cb5_0    defaults
jupyterlab_widgets        3.0.7              pyhd8ed1ab_1    conda-forge
keyring                   23.13.1          py38hecd8cb5_0    defaults
kfp                       1.1.2                    pypi_0    pypi
kfp-notebook              0.17.0                   pypi_0    pypi
kfp-pipeline-spec         0.1.4                    pypi_0    pypi
kfp-server-api            1.3.0                    pypi_0    pypi
krb5                      1.19.4               hdba6334_0    defaults
kubernetes                11.0.0                   pypi_0    pypi
lazy-object-proxy         1.6.0            py38h9ed2024_0    defaults
lcms2                     2.12                 hf1fd2bf_0    defaults
ld64                      530                 h20443b4_25    defaults
ld64_osx-64               530                 h70f3046_25    defaults
ldid                      2.1.5                hc58f1be_3    defaults
lerc                      3.0                  he9d5cce_0    defaults
libarchive                3.6.2                had00e1f_1    defaults
libclang                  14.0.6          default_hd95374b_1    defaults
libclang13                14.0.6          default_habbcc1a_1    defaults
libcurl                   8.1.1                ha585b31_0    defaults
libcxx                    14.0.6               h9765a3e_0    defaults
libdeflate                1.17                 hb664fd8_0    defaults
libedit                   3.1.20221030         h6c40b1e_0    defaults
libev                     4.33                 h9ed2024_1    defaults
libffi                    3.4.4                hecd8cb5_0    defaults
libiconv                  1.16                 hca72f7f_2    defaults
liblief                   0.12.3               hcec6c5f_0    defaults
libllvm14                 14.0.6               h91fad77_3    defaults
libnghttp2                1.52.0               h1c88b7d_1    defaults
libpng                    1.6.39               h6c40b1e_0    defaults
libpq                     12.9                 h1c9f633_3    defaults
libsodium                 1.0.18               h1de35cc_0    defaults
libspatialindex           1.9.3                h23ab428_0    defaults
libssh2                   1.10.0               hdb2fb19_2    defaults
libtiff                   4.5.0                hcec6c5f_2    defaults
libuv                     1.44.2               h6c40b1e_0    defaults
libwebp                   1.2.4                hf6ce154_1    defaults
libwebp-base              1.2.4                h6c40b1e_1    defaults
libxml2                   2.10.3               h930c0e2_0    defaults
libxslt                   1.1.37               h6d1eb0e_0    defaults
llvm-openmp               14.0.6               h0dcd299_0    defaults
lxml                      4.9.2            py38h6c40b1e_0    defaults
lz4-c                     1.9.4                hcec6c5f_0    defaults
markupsafe                2.1.1            py38hca72f7f_0    defaults
matplotlib-inline         0.1.6            py38hecd8cb5_0    defaults
mccabe                    0.7.0              pyhd3eb1b0_0    defaults
minio                     6.0.2                    pypi_0    pypi
mistune                   0.8.4           py38h1de35cc_1001    defaults
more-itertools            8.12.0             pyhd3eb1b0_0    defaults
mypy_extensions           0.4.3            py38hecd8cb5_1    defaults
navigator-updater         0.4.0            py38hecd8cb5_0    defaults
nbclassic                 0.2.5                    pypi_0    pypi
nbclient                  0.5.13           py38hecd8cb5_0    defaults
nbconvert                 5.6.1                    pypi_0    pypi
nbdime                    2.1.0                    pypi_0    pypi
nbformat                  5.7.0            py38hecd8cb5_0    defaults
nbresuse                  0.4.0                    pypi_0    pypi
ncurses                   6.4                  hcec6c5f_0    defaults
nest-asyncio              1.5.6            py38hecd8cb5_0    defaults
nodejs                    18.15.0              hab5cffc_0    defaults
notebook                  6.5.4            py38hecd8cb5_0    defaults
notebook-shim             0.2.2            py38hecd8cb5_0    defaults
nspr                      4.35                 hcec6c5f_0    defaults
nss                       3.89.1               hcec6c5f_0    defaults
numpy                     1.24.3                   pypi_0    pypi
numpydoc                  1.5.0            py38hecd8cb5_0    defaults
oauthlib                  3.1.0                    pypi_0    pypi
openssl                   1.1.1u               hca72f7f_0    defaults
packaging                 23.0             py38hecd8cb5_0    defaults
pandas                    2.0.1                    pypi_0    pypi
pandocfilters             1.5.0              pyhd3eb1b0_0    defaults
papermill                 2.2.2                    pypi_0    pypi
parso                     0.8.3              pyhd3eb1b0_0    defaults
patch                     2.7.6             h1de35cc_1001    defaults
pathlib                   1.0.1                    py38_2    defaults
pathspec                  0.10.3           py38hecd8cb5_0    defaults
pcre                      8.45                 h23ab428_0    defaults
pcre2                     10.42                h9b97e30_0    defaults
perl                      5.34.0               h435f0c2_2    defaults
pexpect                   4.8.0              pyhd3eb1b0_3    defaults
pickleshare               0.7.5           pyhd3eb1b0_1003    defaults
pillow                    9.4.0            py38hcec6c5f_0    defaults
pip                       21.3.1                   pypi_0    pypi
pkginfo                   1.9.6            py38hecd8cb5_0    defaults
pkgutil-resolve-name      1.3.10           py38hecd8cb5_0    defaults
platformdirs              2.5.2            py38hecd8cb5_0    defaults
pluggy                    1.0.0            py38hecd8cb5_1    defaults
ply                       3.11                     py38_0    defaults
poyo                      0.5.0              pyhd3eb1b0_0    defaults
prometheus_client         0.14.1           py38hecd8cb5_0    defaults
prompt-toolkit            3.0.36           py38hecd8cb5_0    defaults
protobuf                  3.14.0                   pypi_0    pypi
psutil                    5.9.2            py38hef030d1_0    conda-forge
ptyprocess                0.7.0              pyhd3eb1b0_2    defaults
pure_eval                 0.2.2              pyhd3eb1b0_0    defaults
py-lief                   0.12.3           py38hcec6c5f_0    defaults
pyasn1                    0.4.8                    pypi_0    pypi
pyasn1-modules            0.2.8                    pypi_0    pypi
pycodestyle               2.10.0           py38hecd8cb5_0    defaults
pycosat                   0.6.4            py38hca72f7f_0    defaults
pycparser                 2.21               pyhd3eb1b0_0    defaults
pydocstyle                6.3.0            py38hecd8cb5_0    defaults
pyflakes                  3.0.1            py38hecd8cb5_0    defaults
pygments                  2.15.1           py38hecd8cb5_1    defaults
pyjwt                     2.4.0            py38hecd8cb5_0    defaults
pylint                    2.16.2           py38hecd8cb5_0    defaults
pylint-venv               2.3.0            py38hecd8cb5_0    defaults
pyls-spyder               0.4.0              pyhd3eb1b0_0    defaults
pyobjc-core               9.0              py38h9205ec4_1    defaults
pyobjc-framework-cocoa    9.0              py38h9205ec4_0    defaults
pyobjc-framework-coreservices 9.0              py38h46256e1_0    defaults
pyobjc-framework-fsevents 9.0              py38hecd8cb5_0    defaults
pyopenssl                 23.2.0           py38hecd8cb5_0    defaults
pyqt                      5.15.7           py38he9d5cce_0    defaults
pyqt5-sip                 12.11.0          py38he9d5cce_0    defaults
pyqtwebengine             5.15.7           py38he9d5cce_0    defaults
pyrsistent                0.18.1           py38hed1de0f_1    conda-forge
pysocks                   1.7.1                    py38_1    defaults
python                    3.8.17               h218abb5_0    defaults
python-dateutil           2.8.2              pyhd3eb1b0_0    defaults
python-fastjsonschema     2.16.2           py38hecd8cb5_0    defaults
python-json-logger        2.0.7            py38hecd8cb5_0    defaults
python-libarchive-c       4.0              py38h50d1736_1    conda-forge
python-lsp-black          1.2.1            py38hecd8cb5_0    defaults
python-lsp-jsonrpc        1.0.0              pyhd3eb1b0_0    defaults
python-lsp-server         1.7.2            py38hecd8cb5_0    defaults
python-slugify            5.0.2              pyhd3eb1b0_0    defaults
python.app                3                py38hca72f7f_0    defaults
python_abi                3.8                      2_cp38    conda-forge
pytoolconfig              1.2.5            py38hecd8cb5_1    defaults
pytz                      2022.7           py38hecd8cb5_0    defaults
pyyaml                    6.0              py38h6c40b1e_1    defaults
pyzmq                     25.1.0           py38hcec6c5f_0    defaults
qdarkstyle                3.0.2              pyhd3eb1b0_0    defaults
qstylizer                 0.2.2            py38hecd8cb5_0    defaults
qt-main                   5.15.2               h51e0635_8    defaults
qt-webengine              5.15.9               h90a370e_4    defaults
qtawesome                 1.2.2            py38hecd8cb5_0    defaults
qtconsole                 5.4.2            py38hecd8cb5_0    defaults
qtpy                      2.2.0            py38hecd8cb5_0    defaults
qtwebkit                  5.212                hbfab81c_5    defaults
readline                  8.2                  hca72f7f_0    defaults
readme-renderer           28.0                     pypi_0    pypi
requests                  2.31.0           py38hecd8cb5_0    defaults
requests-oauthlib         1.3.0                    pypi_0    pypi
requests-toolbelt         0.9.1              pyhd3eb1b0_0    defaults
rfc3339-validator         0.1.4            py38hecd8cb5_0    defaults
rfc3986                   1.4.0                    pypi_0    pypi
rfc3986-validator         0.1.1            py38hecd8cb5_0    defaults
rope                      1.7.0            py38hecd8cb5_0    defaults
rsa                       4.7                      pypi_0    pypi
rtree                     1.0.1            py38hecd8cb5_0    defaults
ruamel.yaml               0.17.21          py38hca72f7f_0    defaults
ruamel.yaml.clib          0.2.6            py38hca72f7f_1    defaults
send2trash                1.8.0              pyhd3eb1b0_1    defaults
setuptools                68.0.0           py38hecd8cb5_0    defaults
sip                       6.6.2            py38he9d5cce_0    defaults
six                       1.16.0             pyhd3eb1b0_1    defaults
smmap                     3.0.4                    pypi_0    pypi
sniffio                   1.3.0              pyhd8ed1ab_0    conda-forge
snowballstemmer           2.2.0              pyhd3eb1b0_0    defaults
sortedcontainers          2.4.0              pyhd3eb1b0_0    defaults
soupsieve                 2.4              py38hecd8cb5_0    defaults
sphinx                    5.0.2            py38hecd8cb5_0    defaults
sphinxcontrib-applehelp   1.0.2              pyhd3eb1b0_0    defaults
sphinxcontrib-devhelp     1.0.2              pyhd3eb1b0_0    defaults
sphinxcontrib-htmlhelp    2.0.0              pyhd3eb1b0_0    defaults
sphinxcontrib-jsmath      1.0.1              pyhd3eb1b0_0    defaults
sphinxcontrib-qthelp      1.0.3              pyhd3eb1b0_0    defaults
sphinxcontrib-serializinghtml 1.1.5              pyhd3eb1b0_0    defaults
spyder                    5.4.3            py38hecd8cb5_1    defaults
spyder-kernels            2.1.0                    pypi_0    pypi
spyder-notebook           0.1.4                      py_0    spyder-ide
sqlite                    3.41.2               h6c40b1e_0    defaults
stack_data                0.2.0              pyhd3eb1b0_0    defaults
strip-hints               0.1.9                    pypi_0    pypi
tabulate                  0.8.7                    pypi_0    pypi
tapi                      1000.10.8            ha1b3eb9_0    defaults
tenacity                  6.3.1                    pypi_0    pypi
terminado                 0.17.1           py38hecd8cb5_0    defaults
text-unidecode            1.3                pyhd3eb1b0_0    defaults
textdistance              4.2.1              pyhd3eb1b0_0    defaults
textwrap3                 0.9.2                    pypi_0    pypi
three-merge               0.1.1              pyhd3eb1b0_0    defaults
tinycss2                  1.2.1            py38hecd8cb5_0    defaults
tk                        8.6.12               h5d9f67b_0    defaults
toml                      0.10.2             pyhd3eb1b0_0    defaults
tomli                     2.0.1            py38hecd8cb5_0    defaults
tomlkit                   0.11.1           py38hecd8cb5_0    defaults
toolz                     0.12.0           py38hecd8cb5_0    defaults
tornado                   6.2              py38hca72f7f_0    defaults
tox                       3.21.2                   pypi_0    pypi
tqdm                      4.65.0           py38h01d92e1_0    defaults
traitlets                 5.7.1            py38hecd8cb5_0    defaults
twine                     3.3.0                    pypi_0    pypi
typing-extensions         4.7.1            py38hecd8cb5_0    defaults
typing_extensions         4.7.1            py38hecd8cb5_0    defaults
tzdata                    2023.3                   pypi_0    pypi
ujson                     5.4.0            py38he9d5cce_0    defaults
unidecode                 1.2.0              pyhd3eb1b0_0    defaults
urllib3                   1.26.16          py38hecd8cb5_0    defaults
virtualenv                20.4.0                   pypi_0    pypi
watchdog                  2.1.6            py38h999c104_0    defaults
wcwidth                   0.2.5              pyhd3eb1b0_0    defaults
webencodings              0.5.1                    py38_1    defaults
websocket-client          0.57.0                   pypi_0    pypi
whatthepatch              1.0.2            py38hecd8cb5_0    defaults
wheel                     0.38.4           py38hecd8cb5_0    defaults
widgetsnbextension        4.0.5            py38hecd8cb5_0    defaults
wrapt                     1.14.1           py38hca72f7f_0    defaults
wurlitzer                 2.1.1                    pypi_0    pypi
xlwings                   0.30.4                   pypi_0    pypi
xz                        5.4.2                h6c40b1e_0    defaults
yaml                      0.2.5                haf1e3a3_0    defaults
yapf                      0.31.0             pyhd3eb1b0_0    defaults
zeromq                    4.3.4                h23ab428_0    defaults
zipp                      3.11.0           py38hecd8cb5_0    defaults
zlib                      1.2.13               h4dc903c_0    defaults
zstandard                 0.19.0           py38h6c40b1e_0    defaults
zstd                      1.5.5                hc035e20_0    defaults
```

</details>

### PR DESCRIPTION
**Pull request by dholth**
### Description

Fix #4949 (avoid conda-build side effects when not using conda-build subcommands). Should improve conda start time when conda-build is installed.

Untested.

### Checklist - did you ...

- [x] Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?
     
- [x] Add / update necessary tests?
- [x] Add / update outdated documentation?

### CODE REVIEW

File: conda_build/__init__.py [hunk 1]
```diff
@@ -12,5 +12,6 @@
     "index",
     "inspect",
     "metapackage",
-    "render" "skeleton",
```
Comment (dholth) [inline conda_build/__init__.py:15]: Whoops.

Do we use this list?

Comment (kenodegard) [reply conda_build/__init__.py:15]: i guess not?

**Comment by kenodegard** [comment]
pre-commit.ci autofix

**Review by kenodegard — APPROVED** [review]
tested this locally and this change prevents the debugging messages from occurring for non-conda-build subcommands

**Pull request merged**